TalkableSDK.podspec

```diff
@@ -0,0 +1,36 @@
+
+Pod::Spec.new do |spec|
+
+  spec.name         = "TalkableSDK"
+  spec.version      = "1.0"
+  spec.summary      = "Talkable SDK makes it easy to integrate Talkable referral functionality into your apps."
+
+  spec.description  = <<-DESC
+  Once integrated you can use the following Talkable capabilities:
+
+  - Display Advocate Share Page (the page itself is built inside Talkable)
+  - Share Offer via:
+    - Email
+    - Facebook
+    - Twitter
+    - SMS
+    - by copying a direct Offer link
+    - Track sales via the App and reward Advocate if a sale was driven thourgh someone’s claim link
+                   DESC
+
+  spec.homepage     = "https://github.com/talkable/ios-sdk"
+  spec.license      = { :type => 'MIT' }
+  spec.author             = "Talkable"
+  spec.social_media_url   = "https://twitter.com/talkable"
+
+  spec.platform     = :ios, "9.0"
+
+  spec.source       = { :git => "https://github.com/talkable/ios-sdk.git", :tag => "#{spec.version}" }
```

Comment:
Looks like we need to add a correspond tag to have it work

Reply:
Yes, we need to add a version tag, so Cocoapods will know which version distribute

Reply:
So please create a pre-release in this branch, update the code and test if it works: https://github.com/talkable/ios-sdk/releases/new
Ask me if you need help